### ISSUE TITLE
documentation for function: network.find_shortest_path

### ISSUE THREAD

**Issue by asu-trans-ai-lab** [jdlph/Path4GMNS#27]
In the tutorial from version 0.8.7, there is a line of code as follows which is confusing.

```python
print('\nshortest path (link id) from node 1 to node 2, '
      +network.find_shortest_path(1, 2, seq_type='link')) 
```

The output for the Chicago network will be

```python
shortest path (link id) from node 1 to node 2, distance: 3.06 mi | path: 1;986;989
```

[Its use case](https://path4gmns.readthedocs.io/en/latest/usecases.html#get-the-shortest-path-between-two-nodes) has no related statement on how to use arg seq_type. API should discuss the parameters of this very important function.

**Comment by asu-trans-ai-lab** [comment]
can we output the results of network.find_shortest_path(1, 2) to agent_path.csv so that plot4gmns can visualize the final paths?

**Comment by jdlph** [comment]
It has been addressed in commit [7238edf](https://github.com/jdlph/Path4GMNS/commit/7238edf59912ee385946b643783e9cf2ec54c561).

1. specify link path or node path in the printout of find_shortest_path().
2. add comments with respect to seq_type in [use cases](https://path4gmns.readthedocs.io/en/latest/usecases.html#get-the-shortest-path-between-two-nodes).

> can we output the results of network.find_shortest_path(1, 2) to agent_path.csv so that plot4gmns can visualize the final paths?

This request will be postponed to a later stage when json is adopted for data exchange between the two packages.
